src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Performance/CSharpPreferHashDataOverComputeHash.Fixer.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.NetCore.Analyzers.Performance;
+
+namespace Microsoft.NetCore.CSharp.Analyzers.Performance
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public sealed class CSharpPreferHashDataOverComputeHashFixer : PreferHashDataOverComputeHashFixer
+    {
+        protected override bool TryGetCodeFixer(SyntaxNode computeHashNode, SyntaxNode bufferArgNode, SyntaxNode nodeToRemove, [NotNullWhen(true)] out ApplyCodeFixAction? codeFixer)
+        {
+            switch (nodeToRemove)
+            {
+                case LocalDeclarationStatementSyntax localDeclarationStatement:
+                    {
+                        codeFixer = (editor, hashDataInvoked) =>
+                        {
+                            editor.ReplaceNode(computeHashNode, hashDataInvoked);
+                            editor.RemoveNode(localDeclarationStatement);
+                        };
+                        return true;
+                    }
+
+                case VariableDeclaratorSyntax variableDeclaratorSyntax:
+                    {
+                        codeFixer = (editor, hashDataInvoked) =>
+                        {
+                            editor.ReplaceNode(computeHashNode, hashDataInvoked);
+                            editor.RemoveNode(variableDeclaratorSyntax);
+                        };
+                        return true;
+                    }
```

Comment:
This code looks duplicated, can't it be a single case? e.g `case LocalDeclarationStatementSyntax or VariableDeclaratorSyntax` and use `editor.RemoveNode(nodeToRemove)`.